RCTComms/comms.py

```diff
@@ -871,8 +910,9 @@ class gcsComms:
     Radio Collar Tracker UDP Interface
     '''
     __BUFFER_LEN = 1024
+    __lock = threading.Lock()
```

Comment:
Should this be a static class scope variable?  Or should it be an instance variable, defined in `__init__`?